packages/react-i18n/src/Provider.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import {I18nContext} from './context';
+import Manager from './manager';
+
+interface Props {
+  manager?: Manager;
+  children: React.ReactNode;
+}
+
+export function Provider({manager, children}: Props) {
```

Comment:
What do you think, do we still need this component? I feel like maybe it's fine to just expose the raw context object directly but I am not sure :S

Reply:
I think the advantage of exposing the raw context instead is that this is a documented pattern for the React Context API. So if you are using context anywhere else in react, this is the same thing.